### PR TITLE
feat(sbb-tag-group): allow other slotted elements than `sbb-tag`s

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/components/sbb-tag-group/sbb-tag-group.tsx [hunk 1]
```diff
@@ -117,17 +125,24 @@ export class SbbTagGroup implements ComponentInterface {
     return Array.from(this._element.querySelectorAll('sbb-tag')) as HTMLSbbTagElement[];
   }
 
+  private _readSlottedElements(): void {
+    this._slottedElements = Array.from(this._element.children).filter(
+      (e): e is HTMLElement => e instanceof window.HTMLElement,
+    );
+  }
+
```
Comment (TomMenga) [inline src/components/sbb-tag-group/sbb-tag-group.tsx:133]: Should it be a getter like `_tags`?
What if the slotted content changes?

**Review by DavideMininni-Fincons — COMMENTED** [review]
It looks good to me; 
should the new case be tested? maybe just changing one sbb-tag in the actual 'renders' test with another sbb- component?

File: src/components/sbb-tag-group/sbb-tag-group.tsx (hunk 1 above)
Comment (DavideMininni-Fincons) [reply src/components/sbb-tag-group/sbb-tag-group.tsx:133]: > What if the slotted content changes?

For content changes there should be the `this._readSlottedElements()` on line 156

File: src/components/sbb-tag-group/sbb-tag-group.e2e.ts
Comment (TomMenga) [inline src/components/sbb-tag-group/sbb-tag-group.e2e.ts:1]: Side note: The tag group has already been migrated to lit, we need to cherry and manually merge the differences

File: src/components/sbb-tag-group/sbb-tag-group.tsx (hunk 1 above)
Comment (TomMenga) [reply src/components/sbb-tag-group/sbb-tag-group.tsx:133]: you're right, soz :/

**Comment by MarioCastigliano** [comment]
Also I think @DavideMininni-Fincons 's suggestion about the tests is a nice idea

File: src/components/sbb-tag-group/sbb-tag-group.e2e.ts
Comment (jeripeierSBB) [reply src/components/sbb-tag-group/sbb-tag-group.e2e.ts:1]: I sadly know :(,but it's kind of a blocker for one consumer, so I decided to implement now

**Comment by jeripeierSBB** [comment]
@MarioCastigliano @DavideMininni-Fincons, absolutely a good idea ;) Added a simple enhancement to the tests

**Pull request merged**